stable/cluster-autoscaler/templates/deployment.yaml

```diff
@@ -72,8 +72,8 @@ spec:
               value: "{{ .Values.spotinst.account }}"
           {{- end }}
           {{- range $key, $value := .Values.extraEnv }}
-          - name: {{ $key }}
-            value: {{ $value }}
+            - name: {{ $key }}
+              value: {{ $value }}
```

Comment:
While you're at it, please make sure the value is quoted. Env vars have to be strings.

Reply:
Quotes are added. Thank you :)